io/plugins/eu.esdihumboldt.hale.io.appschema.test/src/eu/esdihumboldt/hale/io/geoserver/rest/ResourceTestBase.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2018 wetransform GmbH
+ * 
+ * All rights reserved. This program and the accompanying materials are made
+ * available under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution. If not, see <http://www.gnu.org/licenses/>.
+ * 
+ * Contributors:
+ *     wetransform GmbH <http://www.wetransform.to>
+ */
+
+package eu.esdihumboldt.hale.io.geoserver.rest;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Node;
+
+import eu.esdihumboldt.hale.io.geoserver.Resource;
+
+/**
+ * Base class with utility methods for testing the correct serialization of
+ * GeoServer resources.
+ * 
+ * @author Stefano Costa, GeoSolutions
+ */
+public class ResourceTestBase {
+
+	protected static final DocumentBuilderFactory dbf;
```

Comment:
Missing JavaDoc for `protected` declaration

Reply:
Fixed.